src/UnityExtension/Assets/Editor/GitHub.Unity/UI/PUblishView.cs

```diff
@@ -0,0 +1,242 @@
+﻿using System;
+using System.Linq;
+using Octokit;
+using UnityEditor;
+using UnityEngine;
+
+namespace GitHub.Unity
+{
+    class PublishView : Subview
+    {
+        private const string Title = "Publish this repository to GitHub";
+        private const string PrivateRepoMessage = "You choose who can see and commit to this repository";
+        private const string PublicRepoMessage = "Anyone can see this repository. You choose who can commit";
+
+        [SerializeField]
+        private string username;
+        [SerializeField]
+        private string[] owners = { };
+        [SerializeField]
+        private int selectedOwner;
+        [SerializeField]
+        private string repoName = String.Empty;
+        [SerializeField]
+        private string repoDescription = "";
+        [SerializeField]
+        private bool togglePrivate;
+
+        [NonSerialized]
+        private IApiClient client;
+        [NonSerialized]
+        private bool isBusy;
+        [NonSerialized]
+        private string error;
```

Comment:
Could you put this stuff all in one line for readibility? Yeah, I know, R# refuses to do it properly, it's a pain in the ass